### DISCUSSION TITLE
SSH key for beginer

### DISCUSSION THREAD

**Discussion by ghost** [community/community#145843] [Code Security] [answered]
### Select Topic Area

Question

### Body

Hello, everyone.
I just started learning computer programing and I wanna know more about the how to use github ssh-key.
How to generate and how to access private repository using it?

**Comment by ledgerluminary108** [comment, marked as answer]
To generate an SSH key for GitHub, open your terminal and run the command 
`ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
`Replace the email with your GitHub account email. To access a private repository using SSH, first add your public key to your GitHub account, then clone the repository using the SSH URL like this: 
`git clone git@github.com:username/repository.git`
